runelite-client/src/main/java/net/runelite/client/plugins/objectindicators/ObjectIndicatorsPlugin.java

```diff
@@ -180,12 +183,42 @@ public void onGameObjectSpawned(GameObjectSpawned event)
 		}
 	}
 
+	@Subscribe
+	public void onDecorativeObjectSpawned(DecorativeObjectSpawned event) {
+		final WorldPoint worldPoint = WorldPoint.fromLocalInstance(client, event.getDecorativeObject().getLocalLocation());
```

Comment:
I am pretty sure you can just reuse the logic in gameObjectSpawned

Reply:
that is the same logic from the GameObjectSpawned, should I put it in a private method or something?

Reply:
yes

Reply:
The changes have been made